lib/apipie/error_description.rb

```diff
@@ -1,9 +1,11 @@
 module Apipie
 
   class ErrorDescription
-
     attr_reader :code, :description, :metadata
 
+    class UnrecognizedCode < Exception
```

Comment:
We have all other errors at one place her https://github.com/Apipie/apipie-rails/blob/296c097f9acb951149d3a67db69b84eb216bdd7e/lib/apipie/errors.rb. Also, prefer `StandardError`, or in this case even `Apipie::Error` over `Exception`